internal/web/context.go

```diff
@@ -0,0 +1,22 @@
+package web
+
+import (
+	"github.com/labstack/echo/v4"
+	"go.uber.org/zap"
+)
+
+type AppContext struct {
+	echo.Context
+	AppLogger *zap.Logger
+}
```

Comment:
The function `CreateAppContext` can be improved for better clarity and maintainability. Adding explicit types for the returned function parameters can enhance readability, especially for developers unfamiliar with the code context. For example, clearly indicate the types of `next` and `c` in comments if you choose to keep the function signature terse.